pkg/sockets/parser.go

```diff
@@ -125,6 +125,18 @@ func (h *Hub) parseAndExecuteMessage(ctx context.Context, msg []byte, sender str
 		}
 		h.removeFromRealm(Realm(evt.Realm), sender)
 
+	case pb.MessageType_TIMED_OUT:
+		evt, ok := ew.Event.(*pb.TimedOut)
+		if !ok {
+			return errors.New("to unexpected typing error")
+		}
+		// Verify the timeout.
+		err := gameplay.TimedOut(ctx, h.gameStore, sender, evt.GameId)
+		if err != nil {
+			return err
+		}
+		// XXX: send a game over event
```

Comment:
the `TimedOut` function sends the game over event, so i need to remove this comment.